src/components/App/App.tsx

```diff
@@ -57,61 +57,59 @@ const App = (): JSX.Element => {
 
 
   return (
-    <div className="App">
```

Comment:
This looks like a bigger change that it is... I just removed the outermost div since we already have a div ('container') surrounding this.